src/server/deviceDescriptorsSource.json

```diff
@@ -1140,5 +1140,125 @@
     "isMobile": true,
     "hasTouch": true,
     "defaultBrowserType": "chromium"
+  },
+  "High DPI Notebook Google Chrome": {
```

Comment:
There is a professional slang for mobile browsers vs the desktop browsers (https://gs.statcounter.com/platform-market-share/desktop-mobile-tablet).

`Desktop Safari` - always high dpi
`Desktop Chrome`
`Desktop Edge`
`Desktop Firefox`
`Desktop Chrome HiDPI`
`Desktop Edge HiDPI`
`Desktop Firefox HiDPI`

Reply:
implemented it like that. Choose Windows as a OS, since it has the biggest market share. For Safari I kept MacOS.